app/views/sufia/batch_uploads/_form_files.html.erb

```diff
@@ -100,6 +112,18 @@
                 <div class="col-sm-7">
                   <input type="text" class="form-control" name="title[{%=file.id%}]" id="title_{%=file.id%}" value="{%=file.name%}">
                 </div>
+                <label for="resource_type_{%=file.id%}" class="col-sm-5 control-label">Resource Type</label>
+                <div class="col-sm-7">
+                  <select class="form-control resource_type_dropdown" name="resource_type[{%=file.id%}]" id="resource_type_{%=file.id%}" value="{%=file.name%}">
+                    <% Sufia.config.resource_types.each do |type| %>
+                        <option value="<%= type[0] %>"><%= type[1] %></option>
+                    <% end %>
+                  </select>
+                  <!-- TODO: Why is the button drawn for all files? -->
```

Comment:
I cannot get the button to be displayed only for the first file. It shows for all of them. Any ideas?

I can add a separate issue to track this problem if you want to go ahead and merge the current PR in time for the demo tomorrow.